src/render_image.rs

```diff
@@ -264,68 +282,69 @@ impl<'a> Step {
-                            iced::Theme::Dark => ThemeType::Dark,
-                            iced::Theme::Light => ThemeType::Light,
-                            iced::Theme::Custom { .. } => ThemeType::Custom,
-                        }),
-                        ImageStepMessage::ThemeChanged,
-                    );
-                    column.push(element);
-                },
-            );
-        let choose_theme_content = column![choose_theme]
+    pub fn welcome(obj: &Steps) -> Element<ImageStepMessage> {
+        let choose_theme: Column<'_, ImageStepMessage, _> = column![
+            iced::widget::text("Theme:"),
+            pick_list(
+                ThemeType::ALL,
+                Some(match obj.theme {
+                    iced::Theme::Dark => ThemeType::Dark,
+                    iced::Theme::Light => ThemeType::Light,
+                    iced::Theme::Custom { .. } => ThemeType::Custom,
+                }),
+                ImageStepMessage::ThemeChanged
+            ),
+            // .width(Length::Fill),
+        ]
+        .spacing(10);
+
+        let choose_theme_content: Column<'_, ImageStepMessage> = column![choose_theme]
             .spacing(20)
             .padding(20)
             .max_width(600)
             .width(Length::Fill);
 
         unsafe {
             if FOLDER_FOUND {
-                let file_choose_button = button(text("Select folder"))
-                    .on_press(ImageStepMessage::ChooseFolderPath())
-                    .style(theme::Button::Secondary);
+                let file_choose_button: Button<'_, ImageStepMessage> =
+                    button(text("Select folder"))
+                        .on_press(ImageStepMessage::ChooseFolderPath())
+                        .style(theme::Button::Secondary);
                 column![
                     container(row![choose_theme_content
                         .width(Length::Fill)
-                        .align_items(Alignment::Start)]),
-                    container(row![file_choose_button])
+                        .align_items(iced_core::Alignment::Start)]),
+                    file_choose_button
                 ]
+                .into()
             } else {
-                let file_choose_button = button(text("Select folder"))
-                    .on_press(ImageStepMessage::ChooseFolderPath())
-                    .style(theme::Button::Primary);
+                let file_choose_button: Button<'_, ImageStepMessage> =
+                    button(text("Select folder"))
+                        .on_press(ImageStepMessage::ChooseFolderPath())
+                        .style(theme::Button::Primary);
                 column![
                     container(row![choose_theme_content
                         .width(Length::Fill)
-                        .align_items(Alignment::Start)]),
-                    container(row![file_choose_button])
+                        .align_items(iced_core::Alignment::Start)]),
```

Comment:
I've used previously an import for Alignment

Reply:
Ah yes, I think I accidentally removed it while debugging and ended up using the absolute import. I'll consider this refactoring and using better imports + adding documentation in another PR, since there are definitely a lot of  improvements needed, but thanks for highlighting :)